google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -112,4 +122,134 @@ public static BigtableDataClient create(BigtableDataSettings settings) throws IO
   public void close() throws Exception {
     stub.close();
   }
+
+  /**
```

Comment:
The conventional order is for close() to come after the API methods

Reply:
done